i18n/en.ts

```diff
@@ -1,6 +1,43 @@
 import type { Translations } from './translations';
 
 const text: Translations = {
+  Departments: {
+    subLinks: [
```

Comment:
Sorted or not, this is now in a different order than in the other translation files.

Reply:
Sir, may I suggest placing the department at the 0th index and then proceeding to copy and paste whatever content is present in the master? If this is not the requirement, please tell me.

Reply:
I'm confused. What are you trying to say? Copy paste what's in master? Wdym?